### ISSUE TITLE
[p0ng] Game settings not saved on production build

### ISSUE THREAD

**Issue by miketvo** [miketvo/classic-games-godot#14]
Only happens with production build. Cannot see game settings file in `user://` Debug build works fine.

**Comment by miketvo** [comment]
Resolved in #22

**Comment by miketvo** [comment]
# Explanation of what caused the issue:

`ConfigFile.load()` and `ConfigFile.save()` don't play well with the assert statement in production build: If you have code such as:

```gdscript
var config_file: ConfigFile = ConfigFile.new()
assert(config_file.load() == OK, "Cannot load file")
```

or

```gdscript
var config_file: ConfigFile = ConfigFile.new()
assert(config_file.save("res://config.ini") == OK, "Cannot save file")
```

Then in production build, these lines would just not get run (refer to ⁠Godot⁠), and there will be no saving nor loading.
